src/Validation/Validation.php

```diff
@@ -1165,6 +1165,7 @@ public static function uploadedFile($file, array $options = [])
             'maxSize' => null,
             'types' => null,
             'optional' => false,
+            'isUploadedFile' => true
```

Comment:
IMO this suggests that you're telling a file is uploaded and a validator actually shouldn't check it.

Maybe `checkUploadedFile` would better describe what it does?